### DISCUSSION TITLE
Possible livecheck for nexus formula

### DISCUSSION THREAD

**Discussion by marble-sh** [Homebrew/discussions#2503] [Writing Formulae/Casks] [answered]
If you preform the following curl command you can get the latest version of nexus:

```
❯ curl -sI -X CONNECT 'https://download.sonatype.com/nexus/3/latest-mac.tgz'
HTTP/2 302
date: Mon, 22 Nov 2021 01:11:06 GMT
content-type: text/html
content-length: 138
location: https://sonatype-download.global.ssl.fastly.net/repository/downloads-prod-group/3/nexus-3.36.0-01-mac.tgz
server: nginx
surrogate-control: max-age=1209600
```

I was wondering if there was a good way to do this sort of thing in a livecheck block. If so I'd be able to update the nexus formula to support livechecks.

https://github.com/Homebrew/homebrew-core/blob/dd9618634bda1ca72dec3a0dfe87be8ff0ce90b0/Formula/nexus.rb

**Comment by SMillerDev** [comment, marked as answer]
It already has a livecheck block that works?

**Reply by marble-sh** [reply]
I was referring to this comment:

```
  # As of writing, upstream is publishing both v2 and v3 releases. The "latest"
  # release on GitHub isn't reliable, as it can point to a release from either
  # one of these major versions depending on which was published most recently.
```

I might be mistaken but I think the live check block only looks for what's already defined, and manual updates need to be made to the file?

I might have misunderstood how livecheck works though.

**Reply by SMillerDev** [reply]
No, that is just an explanation why this regex is used for the livecheck block. It's working fine.
